app/components/PickBakerStakeAmount.tsx

```diff
@@ -45,7 +45,7 @@ function useCapitalBoundCheck(
     accountInfo: AccountInfo | undefined,
     stake: string
 ): { showWarning: true; limitAfterUpdate: bigint } | { showWarning: false } {
-    const rewardStatus = useAsyncMemo(getRewardStatusLatest);
+    const rewardStatus = useAsyncMemo(getRewardStatusLatest, noOp, []);
```

Comment:
What does this have to do with the fix?

Reply:
I saw this and changed it to not run every time (because the dep array was undefined)